### DISCUSSION TITLE
What is the best way to display menu tree

### DISCUSSION THREAD

**Discussion by PubliAlex** [OFFLINE-GmbH/oc-boxes-support#22] [Help] [answered]
Hello,

I didn't find in the documentation, what is the best way to display the boxes pages menu tree ? I didn't find component to quickly display it.

Best regards,

**Comment by tobias-kuendig** [comment, marked as answer]
You can install the RainLab.Pages plugin and put specific pages or a whole page tree in the menu component.

Another option would be to simply fetch all pages and generate the menu yourself:

```php
$pageTree = \OFFLINE\Boxes\Models\Page::standalonePages()->get()->toNested(false);
```
